index.html

```diff
@@ -3,173 +3,150 @@
-								Flux</i>. It's going to be a great opportunity to learn from the community.<br><br>
-							</li>
-							<li>
-								August'16 : I'll start my bachelor's degree in Computer Science from International Institute of Information Technology,
-								Bhubaneswar. This is a big step in my life, Kinda excited but anxious at the same time. Let's see what this holds for me.
-							</li>
-						</ul>
-						
-					
-					</p>
+				<div class="row">
+					<div class="col-12 mt-5 text-left">
+						<p class="head blog">Blogs :</p>
+					</div>
 				</div>
-			</div>
-		</section>
-
-		<!-- Social -->
-		<section id="social" class="section section-3">
-			<div class="container">
-				<div class="has-text-centered">
-					<h3 class="title is-3">Let's Socialize</h3>
+				<div class="row mt-4">
+					<div class="col-12 text-left">
+						<p class="head"> ---- Click <a href="blog/index.html">here</a> for GSoC'18 related blogs.</p>
+					</div>
 				</div>
-				<div class="social-container columns">
-					<div class="column is-6 has-text-centered">
-						<a target="_blank" href="https://www.linkedin.com/in/ayush-shridhar-23480b130/"><img class="" src="assets/img/linkedin.png"></a>
+				<div class="row">
+					<div class="col-12 mt-4 text-left">
+						<p class="head blog">Timeline :</p>
 					</div>
-					<div class="column is-6 has-text-centered">
-						<a target="_blank" href="https://www.facebook.com/ayush.shridhar.is.batman?ref=br_rs"><img class="" src="assets/img/facebook.png"></a>
+				</div>
+				<div class="row mt-3">
+					<div class="col-12 text-left">
+						<p class="head">
+							- September'19: It's official: I'll most probably be visiting the Australian National University
+							for 6 months, starting Dec'19, working on a test suite for driverless car research. Exciting times
+							indeed!<br><br>
+							- September'19: Successfully completed GSoC'19 with Mozilla. I've published a brief blog about my work on medium :
+							<a target = "_blank" href="https://medium.com/@ayush1999/gsoc19-mozilla-a185b66d9564">here</a>.<br><br>
+							- August'19 : Back in India! Heading to my university at the moment. It was a great experience
+							working in UCL. I hope to continue contributing as and when I get time.<br><br>
+							- May'19: Very rarely do I have more than one good news in a single month. I guess this is one of those rare
+							occasions. I got my talk selected in JuliaCon'19 in University of Maryland, Baltimore. It's great to get
+							this opportunity to showcase my work to a wide audience. Hopefully my visa gets approved (which is doubtful,
+							since I'll be applying from UK,which isn't my home country.). Let's see how this goes. <br><br>
+							- May'19 : Some good news: I got my Google Summer of Code project selected. This time, I'll be working under the 
+							Mozilla organization on the <i>bugbug</i> project. I'll be tough managing it with my project in UCL, let's 
```

Comment:
nit: `I'll -> It'll`

Reply:
No href link provided for projects section in navbar.

Reply:
Link them to https://github.com/ayush-1506?tab=repositories for now.

Reply:
Replace projects with `Blog` in navbar. href for this should link to `/blog/main.html`